dart/lib/src/sentry_exception_factory.dart

```diff
@@ -6,17 +6,21 @@ import 'sentry_stack_trace_factory.dart';
 
 /// class to convert Dart Error and exception to SentryException
 class SentryExceptionFactory {
+  final SentryOptions _options;
+
   SentryStackTraceFactory _stacktraceFactory;
 
+  SentryStackTraceFactory get stacktraceFactory => _stacktraceFactory;
+
   SentryExceptionFactory({
     SentryStackTraceFactory stacktraceFactory,
     @required SentryOptions options,
-  }) {
-    if (options == null) {
+  }) : _options = options {
+    if (_options == null) {
       throw ArgumentError('SentryOptions is required.');
     }
 
-    _stacktraceFactory = stacktraceFactory ?? SentryStackTraceFactory(options);
+    _stacktraceFactory = stacktraceFactory ?? SentryStackTraceFactory(_options);
```

Comment:
maybe `SentryStackTraceFactory` should be required too and throw ArgumentError if null, as SentryClient will take care of create a instance of SentryStackTraceFactory

Reply:
✅